pkg/ebpf/events_pipeline_bench_test.go

```diff
@@ -247,8 +246,8 @@ func BenchmarkNewEventObject(b *testing.B) {
 					UserID:                int(ctx.Uid),
 					MountNS:               int(ctx.MntID),
 					PIDNS:                 int(ctx.PidID),
-					ProcessName:           string(bytes.TrimRight(ctx.Comm[:], "\x00")),
-					HostName:              string(bytes.TrimRight(ctx.UtsName[:], "\x00")),
+					ProcessName:           string(utils.TrimTrailingNUL(ctx.Comm[:])),
```

Comment:
Is it possible this will keep the backing string? Long term, shouldn't we make a copy here?

Reply:
As far we convert to new strings `string(...)`, we're safe.

Reply:
Ah, I thought it was just a soft cast in this case. Alright then.